backend/register_lambda.py

```diff
@@ -0,0 +1,27 @@
+import boto3
+
+dynamodb = boto3.client('dynamodb')
+
+def lambda_handler(event, context):
+    params = event['queryStringParameters']
+    email = params['email']
+    password = params['password']
+
+    dynamodb.put_item(
+        TableName='Emails',
+        Item={
+            'email': {'S': email},
```

Comment:
Why are we storing a password here? Wouldn't we need to store an email and a corresponding Dropbox token? The only place we would need password is for the main account we use to poll which users aren't registering

Reply:
Let's chat about this offline! I was a little confused when I set this up, so happy to update this once we can confirm the expected behavior.